scripts/newinstall.sh

```diff
@@ -919,10 +919,9 @@ n8l::generate_loader_bash() {
 	if [[ -n $miniconda_path ]]; then
 		local cmd_setup_miniconda
 		cmd_setup_miniconda="$(cat <<-EOF
-			export PATH="${miniconda_path}/bin:\${PATH}"
 			export LSST_CONDA_ENV_NAME=\${LSST_CONDA_ENV_NAME:-${LSST_CONDA_ENV_NAME}}
 			# shellcheck disable=SC1091
-			source activate "\$LSST_CONDA_ENV_NAME"
+			source "${miniconda_path}/bin/activate" "\$LSST_CONDA_ENV_NAME"
```

Comment:
At some point aren't we supposed to be switching to `conda activate` ?

Reply:
I'll leave that modernization to someone else.